### DISCUSSION TITLE
next / image questions...

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#40280] [Help]
Of course I am getting another error in lighthouse related to images...  but question is this..   

Can I use a 1200 x 630  png image and specify size of 400x 210 with next/image ??  Is that a problem?  .. It is the same aspect ratio...

For some reason lighthouse is telling is a low resolution image and it is expecting it to be 600 x 315

**Comment by gaston-flores** [comment]
No, it shouldn't be an issue. Which layout are you using? For some layouts [you need to pass the rendered size, for some the original size.](https://nextjs.org/docs/api-reference/next/image#width). That might be what's causing it to be rendered at a smaller size.

**Comment by Elindo586** [comment]
I want smaller size...  

The picture original size is 1200 x 630... but the code I have below makes it smaller, which is OK with me..

Only thing is that lighthouse is expecting it to be 600 x 315

```
</div>
                  <Image
                    src={item.picture}
                    alt={item.alt}
                    width={400}
                    height={210}
                  ></Image>
                  <div>

```

**Reply by gaston-flores** [reply]
Your container is likely larger than the image itself. This isn't really an issue, but you can try setting `layout="fixed"` on Image to get rid of the lighthouse error. The image will be resized to be exactly 400 by 210.

**Comment by Elindo586** [comment]
hmm...

where would the `layout = "fixed"` would go?   Is that within the image tag properties?

Lighthouse is going insane... I got another picture of 263x191  with the very same specifications in the tag, and lighthouse is telling me that its expected to be  395x287...

I got the layout = fixed and still gave me the same error... 

```
<div className="d-flex justify-content-center">
              <Image
                src="/images/rexrothpump-1.png"
                alt="piston-pump"
                width={263}
                height={191}
                layout="fixed"
              />

```


BTW... the previous picture on the original post I made it as big as they wanted it so the error went away, and the pictures size still looking good.... but this last one I surely want to leave it at its actual size and not make it bigger.

**Comment by Elindo586** [comment]
Ok... layout worked... thanks...

**Comment by Elindo586** [comment]
Ok... if I do `layout="fixed"`  it fixes the error on the desktop browser, but it distorts the image on a mobile device..

Apparently google wants bigger images... which is silly because what if you just want or have an smaller image?

I took this post to GoogleChrome/lighthouse see what recommendations they have.
